package.json

```diff
@@ -59,8 +59,8 @@
     "test": "run-s test:**",
     "test:snapshots": "jest --testPathPattern=\"storyshots.test.js\" --silent",
     "test:snapshots:update": "jest --testPathPattern=\"storyshots.test.js\" -u --silent",
-    "test:e2e": "npm run build:app && testcafe chrome:headless ./tests/e2e/**/*.js -S -s artifacts/screenshots --reporter html:artifacts/screenshots/report.html --assertion-timeout 10000 --selector-timeout 500000",
```

Comment:
I remember that we had to add `npm run build:app &&` in both for local reasons. Without it it was randomly failing. I worked with Jen on that. Did you check that locally it doesn't fail?

Reply:
For local development you will need to have a local build of the app yeah. I didn't check locally but as we point to `public/` in the e2e createServer method for this testing suite this is quite straight forward.

I will add something to the readme regarding having to have a local build